### ISSUE TITLE
Eco Benefits Extrapolation Inaccuracies

### ISSUE THREAD

**Issue by jmorrison1847** [OpenTreeMap/otm-core#2367]
1. Start with a map of empty planting sites, e.g. https://www.opentreemap.org/walthamdemo/map/
![screen shot 2015-12-03 at 10 14 45 am](https://cloud.githubusercontent.com/assets/12401491/11564343/b27a1f6e-99a6-11e5-9aa0-315b6c4568cf.png)
2. Edit a single planting site by adding a species and diameter.
![screen shot 2015-12-03 at 10 15 42 am](https://cloud.githubusercontent.com/assets/12401491/11564379/dbd45f96-99a6-11e5-9573-114bf279d87f.png)
3. Total eco benefits will display as a disproportionately large number based on 1 tree out of the total
![screen shot 2015-12-03 at 10 16 16 am](https://cloud.githubusercontent.com/assets/12401491/11564397/f8295fa2-99a6-11e5-9a16-495b7939414f.png)

**Comment by RickMohr** [comment]
I think this is as intended. We know there are 849 trees. Since we don't have a species/diameter for all of them we extrapolate based on the species/diameters we do know. In this case we only know one species/diameter so we extrapolate that they are all the same. In practice this is an unlikely case. It's more common to have a lot of data so the extrapolation has more meat.

**Comment by maurizi** [comment]
@jmorrison1847 Did you start with a map of empty planting sites, or with a map of _trees_ without a species and diameter?  According to the counts, all of the dots on the map are trees.

**Comment by jmorrison1847** [comment]
This makes sense! Thanks @maurizi and @RickMohr for the explanation. I'll change the title to more accurately reflect the confusion I was having.

**Comment by RickMohr** [comment]
@jmorrison, it's explicit but very small. See the line at the bottom of your step 3 image saying "Based on 1 out 849 total trees"?

**Comment by jwalgran** [comment]
I would like to propose linking to a "methodology" page that explains our eco calculation process.

**Comment by dboyer** [comment]
I agree with @jwalgran that a methodology page would be useful. This could likely also include a few resources on i-Tree.
